src/ui-components/IdentityBadge.tsx

```diff
@@ -53,6 +61,7 @@ const IdentityBadge = ({ className, address, identity, flags }: {className?: str
 		{identity?.web && <li><span className='desc'>web:</span>{identity.web}</li>}
 		{flags?.isCouncil && <li><span className='desc'>Council member</span><CouncilEmoji/></li>}
 		{<li><span className='desc'><a href={`https://polkaverse.com/accounts/${address}`} target='_blank' rel='noreferrer'>Polkaverse Profile</a></span></li>}
+		{web3Name && <li><span className='desc'><a href={`https://w3n.id/${web3Name}`} target='_blank' rel='noreferrer'>Web3 Name Profile</a></span></li>}
```

Comment:
Pleae check this copy with product team and if we have to show DID for kilt only or all parachains